src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -322,6 +322,28 @@ def select_frame(self, locator):
 
         Key attributes for frames are `id` and `name.` See `introduction` for
         details about locating elements.
+        
+        See `Unselect Frame` to cancel the frame selection and return to the Main frame.
+        
+        Please note that the frame search always start from the document root or main frame.
+        
+        Examples:
+        | # Two frame elements with equal names | | |
+        | Select Frame   | xpath=`/``/`frame[@name='left']`/`iframe[@name='left'] | # Selects the 'left' iframe |
```

Comment:
Would it be better just say: 
```
| Select Frame | xpath=//iframe[@name='left'] | # Selects the 'left' iframe |
```

Reply:
Yes, if we do not look at the rest of the example. Then we select by name the first found frame without xpath. The first example is to show the structure.

Reply:
In this  KW it is not good idea to teach people to select similar elements from the DOM. Example must be clear and easy to understand.